iothub/device/src/ModuleClient.cs

```diff
@@ -349,14 +349,14 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
         public Task SendEventAsync(Message message, CancellationToken cancellationToken) => this.internalClient.SendEventAsync(message, cancellationToken);
 
         /// <summary>
-        /// Sends a batch of events to device hub
+        /// Sends a batch of events to device hub. Requires AMQP over WebSockets.
         /// </summary>
```

Comment:
AMQP or AMQP over websocket.

@abhipsaMisra - Do we have E2E for this? If not can you add?.

Reply:
Currently we don't have any E2E for SendEventBatchAsync. Tracked in backlog.